atelier-core/src/io/mod.rs

```diff
@@ -65,6 +65,16 @@ pub trait ModelReader {
     fn read(&mut self, r: &mut impl std::io::Read) -> ModelResult<Model>;
 }
 
+///
+/// Trait implemented to build a model from multiple input sources
+///
+pub trait ModelBuilder {
```

Comment:
I don’t know why this is necessary. This should all be done by the model assembler crate. This doesn’t belong in core.

Reply:
reverted this change